src/cli.c

```diff
@@ -117,6 +118,7 @@ GOptionEntry opt_entries[] = {
 	 "Path to the socket where the seccomp notification fd is received", NULL},
 	{"seccomp-notify-plugins", 0, 0, G_OPTION_ARG_STRING, &opt_seccomp_notify_plugins,
 	 "Plugins to use for managing the seccomp notifications", NULL},
+	{"seccomp-notify-allowed", 0, 0, G_OPTION_ARG_NONE, &opt_seccomp_notify_allowed, "Allow seccomp notify annotations", NULL},
```

Comment:
Does the man page need updating for the new option?

(If this didn’t work for … years, my first instinct would be to delete the code wholesale, but I know nothing of the history.)

Reply:
I don't know much about the history either. @giuseppe ?

Reply:
this was added to experiment with crun-style seccomp notification plugins: https://github.com/containers/crun/blob/main/contrib/seccomp-notify-plugin-example/full.c

Not sure if anyone is using it, we didn't follow up to expose it in Podman because using them is quite difficult

Reply:
but why do we need opt_seccomp_notify_allowed?

If they are specified in `opt_seccomp_notify_plugins` we use them, otherwise not.  Why another bool?

If anything, it should be a separate commit

Reply:
Ok, I removed the `seccomp-notify-allowed` as `opt_seccomp_notify_plugins` already handles that.